static/src/javascripts/projects/common/modules/commercial/dfp/dfp-obj.js

```diff
@@ -0,0 +1,4 @@
+define(function () {
```

Comment:
I think it's harder to understand. So if I want to use the `dfp` map, it's not clear what the module definition is, because the map assignements are spread across a number of modules. This makes it hard to read. Also, there is no rjs compile-time guarantees that the `dfp` is actually populated when a client uses it. That's not clear from the code structure either.